src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -908,13 +908,13 @@
     <value>Generate missing operators</value>
   </data>
   <data name="OverrideEqualsOnOverloadingOperatorEqualsCodeActionTitle" xml:space="preserve">
-    <value>Override object.Equals</value>
+    <value>Override 'object.Equals'</value>
```

Comment:
Sometimes we use `object` and sometimes `Object`, do we want to be consistent? I'd vote for the alias `object` as it is the recommended use in code.